src/main/webapp/WEB-INF/web.xml

```diff
@@ -99,6 +99,33 @@
         </web-resource-collection>
     </security-constraint>
 
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>sd-document-creation</web-resource-name>
+            <description>SmartDocuments' Create Document callback</description>
+            <url-pattern>/rest/document-creation/smartdocuments/callback/*</url-pattern>
+            <http-method>POST</http-method>
+        </web-resource-collection>
+    </security-constraint>
+
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>static</web-resource-name>
+            <description>SmartDocuments' wizard result pages</description>
+            <url-pattern>/static/*</url-pattern>
```

Comment:
This is all static content, that's broader than the description suggests

Reply:
Yeah, this is a bit confusing. If it really just SD result pages then I would make that clear in the URL. E.g.. `/static/smartdocuments/*` or something like that.

Reply:
Or.. change the description because it seems to be just about all public pages? In which case maybe `/public/*` is better?

Reply:
Allowed only `/static/smart-documents-result.html`